tests/integration/cattletest/core/test_svc_discovery.py

```diff
@@ -12,6 +12,11 @@ def nsp(super_client, sim_context):
     return nsp
 
 
+@pytest.fixture(scope='module')
```

Comment:
What is your intention in making this scoped to the module? Because by making it module scope, you're saying "Run this once for the module and cache and re-use the result", right?

Also, really, you aren't using it as a fixture, you're just calling it like a normal function, so you can probably just drop the fixture decorator all together.